main/include/eudaq/LogSender.hh

```diff
@@ -17,6 +18,7 @@ namespace eudaq {
       LogSender();
       ~LogSender();
       void Connect(const std::string & type, const std::string & name, const std::string & server);
+      void disconnect();
```

Comment:
Minor style question: should we capitalize this function name as all others are? `Disconnect()`?